cleverpush/src/main/res/values/styles.xml

```diff
@@ -25,4 +25,7 @@
     <style name="alertDialogTheme" parent="ThemeOverlay.AppCompat.Dialog.Alert">
     </style>
 
+    <style name="app_banner" parent="Theme.MaterialComponents.Light.NoActionBar">
```

Comment:
Let's rename to `cleverpush_app_banner_theme`

Reply:
ok